.changeset/rich-poems-retire.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Love this. Will add these to our core release notes as well until we have the repositories merged 👍